types/rdf-utils-dataset/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for rdf-utils-dataset 1.1
+// Project: https://github.com/rdf-ext/rdf-utils-dataset
+// Definitions by: Chris Wilkinson <https://github.com/thewilkybarkid>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { DatasetIndexed } from 'rdf-dataset-indexed/dataset';
+import { BaseQuad, Term } from 'rdf-js';
+
+declare function resource<D extends DatasetIndexed<BaseQuad, BaseQuad>>(input: D, subject: Term): D;
+
+export { resource };
```

Comment:
It's still not the same 😓 
```suggestion
export = resource;
```

Reply:
That doesn't match https://github.com/rdf-ext/rdf-utils-dataset/blob/6039ad5edad37ac031c0423083239fbcdeedc9be/index.js#L1-L3 though? I'm still genuinely confused by JS exports/imports...

Reply:
Oh 🤬. My mistake too. CJS + ES modules is a mess. TypeScript's magical compilation options make it even more confusing.

Commonjs should be exported with the [`export =`](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require) because its semantics are different than `export default`. Looking good now